go.mod

```diff
@@ -1,6 +1,6 @@
 module go.viam.com/utils
 
-go 1.18
+go 1.20
```

Comment:
@abe-winter I'm introducing a go 1.20 feature in this change - which CI image should I be building with to support this?

Reply:
switch your base image from `canon` to `rdk-devenv`

Reply:
discussed offline, since this library doesn't need most of the dependencies provided by `rdk-devenv`, I ended up using a generic `ubuntu-latest` image with `setup-go/-js/-protoc` instead: https://github.com/viamrobotics/goutils/pull/245